activation/handler_test.go

```diff
@@ -431,15 +443,15 @@ func TestHandler_SyntacticallyValidateAtx(t *testing.T) {
 		atxHdlr := newTestHandler(t, goldenATXID)
 		require.NoError(t, atxs.Add(atxHdlr.cdb, prevAtx))
 
-		challenge := newChallenge(prevAtx.Sequence+1, prevAtx.ID(), prevAtx.ID(), currentLayer.GetEpoch(), nil)
-		atx := newAtx(
-			t,
-			sig,
-			challenge,
-			&types.NIPost{},
-			110,
-			types.GenerateAddress([]byte("aaaa")),
-		) // numunits increased from 100 to 110 between atx and prevAtx
+		challenge := types.NIPostChallenge{
+			Sequence:       prevAtx.Sequence + 1,
+			PrevATXID:      prevAtx.ID(),
+			PublishEpoch:   currentLayer.GetEpoch(),
+			PositioningATX: prevAtx.ID(),
+			CommitmentATX:  nil,
+		}
```

Comment:
Is it an improvement? The original `newChallenge()` version was more concise.

Reply:
It is exactly the same? The difference being that I don't know which parameter encoded which field in the challenge without navigating to the function definition.

Reply:
>It is exactly the same?

It now occupies 7 lines instead of 1 so it's not exactly the same.

>The difference being that I don't know which parameter encoded which field in the challenge without navigating to the function definition.

That's fair :+1: